typeid/typeid-go/prefix.go

```diff
@@ -22,6 +22,12 @@ type AnyID struct {
 	TypeID[AnyPrefix]
 }
 
+type DynamicPrefix string
```

Comment:
Why do you need this? (how is it different from the Any Prefix?). I'm tempted to get rid of it unless I understand the use case.

Reply:
oops, not used. I was just messing around with alternatives. Will rm